Foundation/NSDecimal.swift

```diff
@@ -153,3 +494,30 @@ public func NSDecimalPower(_ result: UnsafeMutablePointer<Decimal>, _ number: Un
 public func NSDecimalMultiplyByPowerOf10(_ result: UnsafeMutablePointer<Decimal>, _ number: UnsafePointer<Decimal>, _ power: Int16, _ roundingMode: NSDecimalNumber.RoundingMode) -> NSDecimalNumber.CalculationError { NSUnimplemented() }
 
 public func NSDecimalString(_ dcm: UnsafePointer<Decimal>, _ locale: AnyObject?) -> String { NSUnimplemented() }
+
+// == Internal (Swifty) functions ==
+
+extension Decimal {
+    fileprivate var isCompact: Bool {
+        get {
+            return _isCompact != 0
+        }
+        set {
+            if newValue {
+                _isCompact = 1
+            } else {
+                _isCompact = 0
+            }
```

Comment:
Yes, it could be but I don't know if there's aversion to using a ternary operator.